xmake/rules/qt/moc/xmake.lua

```diff
@@ -53,7 +53,7 @@ rule("qt.moc")
         local flags = {}
         table.join2(flags, compiler.map_flags("cxx", "define", target:get("defines")))
         table.join2(flags, compiler.map_flags("cxx", "includedir", target:get("includedirs")))
-        table.join2(flags, compiler.map_flags("cxx", "sysincludedir", target:get("sysincludedirs")))
+        table.join2(flags, compiler.map_flags("cxx", "includedir", target:get("sysincludedirs")))
```

Comment:
why? we should use sysincludedir here, because sysincludedir will be mapped to /Iexternal for msvc